bin/plugin/commands/build-plugins.js

```diff
@@ -0,0 +1,200 @@
+ */
+exports.handler = async () => {
+	const pluginsFile = path.join( '.', 'plugins.json' );
+	fs.readFile( pluginsFile, 'utf8', ( err, jsonString ) => {
+		if ( err ) {
+			log(
+				formats.error(
+					`Error reading file from disk: "${ err }"`
+				)
+			);
+		}
+
+		try {
+			const plugins = JSON.parse( jsonString );
+			for ( const moduleDir in plugins ) {
+				const pluginVersion = plugins[ moduleDir ]?.version;
+				const pluginSlug = plugins[ moduleDir ]?.slug;
+
+				if ( ! pluginVersion || ! pluginSlug ) {
+					log(
+						formats.error(
+							'The given module configuration is invalid, the module JSON object is missing the "version" and/or "slug" properties, or they are misspelled.'
+						)
+					);
+					return;
+				}
+
+				try {
+					// Copy module files from the folder.
+					const modulePath = path.join( '.', 'modules/' + moduleDir );
+					const buildModulePath = path.join( '.', 'build/' + pluginSlug );
+					try {
+						// Clean up build module files directory.
+						fs.rmSync( buildModulePath, { force: true, recursive: true } );
+
+						fs.copySync( modulePath, buildModulePath, { overwrite: true } );
+					} catch ( copyError ) {
+						log(
+							formats.error(
+								`Error copying plugin file: "${ copyError }"`
+							)
+						);
+						continue;
+					}
+
+					// Update file content.
+					updatePluginHeader( {
+						key: moduleDir,
+						slug: pluginSlug,
+						version: pluginVersion,
+						pluginPath: buildModulePath,
+					} );
+
+					// Update text domain.
+					updateModuleDetails( {
+						pluginPath: buildModulePath,
+						regex: '[\']performance-lab[\']',
+						result: `'${ pluginSlug }'`,
+					} );
+
+					// Update `@package`.
+					updateModuleDetails( {
+						pluginPath: buildModulePath,
+						regex: '@package\\s{1,}performance-lab',
+						result: '@package ' + pluginSlug,
+					} );
+				} catch ( error ) {
+					log(
+						formats.error(
+							`${ error }`
+						)
+					);
+				}
+			}
+		} catch ( jsonError ) {
+			log(
+				formats.error(
+					`Error parsing JSON string: "${ jsonError }"`
+				)
+			);
+		}
+	} );
+};
+
+/**
+ * Updates the `load.php` file header information.
+ *
+ * @param {[]Settings} settings Plugin settings.
+ */
+async function updatePluginHeader( settings ) {
+	const { key, version, slug, pluginPath } = settings;
+	// Plugin root `load.php` file content.
+	const rootLoadFile = path.join( '.', 'load.php' );
+	const rootLoadFileContent = fs.readFileSync( rootLoadFile, 'utf-8' );
+	// Specific module `load.php` file content.
+	const buildLoadFile = path.join( pluginPath, 'load.php' );
+	const buildLoadFileContent = fs.readFileSync( buildLoadFile, 'utf-8' );
+
+	const rootFileHeader = await getModuleHeader( rootLoadFileContent );
+	const buildFileLoad = await getModuleHeader( buildLoadFileContent );
+
+	// Get module header data.
+	const { name, description } = await getModuleData( buildLoadFileContent );
```

Comment:
You already get the header above, you should be able to extract the module data from that, which is faster.
```suggestion
	const { name, description } = await getModuleDataFromHeader( buildFileLoad );
```